### ISSUE TITLE
Unable to authorize account

### ISSUE THREAD

**Issue by tetlowgm** [Backblaze/B2_Command_Line_Tool#316]
I've just got the 0.7.0 version of the client installed on a fresh box (FreeBSD) and I get the following error message when trying to authorize my account. Any thoughts?

$ b2 authorize-account XXXXXXXXX XXXXXXXXXXXXXXXXXXXXXXXXXX
Using https://api.backblazeb2.com
ERROR: unable to authorize account: Unknown error: error('Error -3 while decompressing: invalid distance too far back',)

**Comment by ppolewicz** [comment]
Sounds like an issue with openssl. Please run it again with `--debugLogs` flag right after authorize-account and share the contents of `b2_cli.log`.

**Comment by tetlowgm** [comment]
Take a look.

[b2_cli.log.txt](https://github.com/Backblaze/B2_Command_Line_Tool/files/718416/b2_cli.log.txt)

**Comment by ppolewicz** [comment]
I am not entirely sure, but it sort of looks like the failure is related to python unsuccessfully trying to use a database of timezones on your system.

**Comment by tetlowgm** [comment]
Okay, I finally found the time to dig back into this. It appears on FreeBSD the current version of the py-dateutil package is running dos2unix on the dateutil-zoneinfo.tar.gz, corrupting the file. I've fixed it on my local system and confirms everything now works. I've additionally reported the bug to the port maintainer for the py-dateutil package. Hopefully it'll be fixed upstream shortly.

Thanks for the help in the diagnosis. Took a bit of digging to figure out the root cause. Glad to see it wasn't anything on your side.
